CRX Known Issues updates (#4240)
GoogleChrome/developer.chrome.com: site/en/docs/extensions/mv3/known-issues/index.md

**Tracking issue:** [1247690][]

> ## Fixed but not yet in Stable {: #resolved }

Reviewer: This page is already pretty heavy with headings. Here's my suggestion. Move `onAuthRequired` to the top of the Capabilities section and replace the estimted timeline with "**Shipped:** 108".

Author: Ack.

### webRequest.onAuthRequired events {: #webrequest-onauthrequired }

> **Summary:** User-installed Manifest V3 extensions cannot currently intercept `webRequest.onAuthRequired` events, since the `webRequestBlocking` permission is restricted to policy-installed extensions. Chrome will provide extensions with a way to supply credentials for authentication requests in Manifest V3.

Reviewer: >Chrome will provide extensions with a way to supply credentials for authentication requests in Manifest V3.

This line looks like it needs to be updated. Also, we should write new material to describe that. Who can give me a brain dump about it?

Author: @rdcronin implemented this. Here are a few relevant resources. I think I can answer most of the questions you're likely to have about this.

- [Issue 1135492](https://bugs.chromium.org/p/chromium/issues/detail?id=1135492)
- [Design doc](https://docs.google.com/document/d/1iNonK6pgl9U5BTELR3iumfOU6TbirmC--a3GkIjUbfc/edit#heading=h.uoiyzkmb4b5l)
- [Code review](https://crrev.com/c/3923277)

Reviewer: I've opened [a ticket](https://github.com/GoogleChrome/developer.chrome.com/issues/4442) so that we can cover this in a separate PR.

Author: Resolving this discussion as it's being tracked in #4442.

> **Summary:** User-installed Manifest V3 extensions cannot currently intercept `webRequest.onAuthRequired` events, since the `webRequestBlocking` permission is restricted to policy-installed extensions. Chrome will provide extensions with a way to supply credentials for authentication requests in Manifest V3.

Reviewer: ```suggestion
**Summary:** User-installed Manifest V3 extensions cannot currently intercept `webRequest.onAuthRequired` events, since the `webRequestBlocking` permission is restricted to policy-installed extensions. Chrome provides extensions with a way to supply credentials for authentication requests in Manifest V3.
```

Author: Ack.

Reviewer: Please commit this change. We're providing a way in the future. We're providing it now.

Reviewer: I see. This got picked up in another correction. Never mind.

**Estimated timeline:** Slated for release in Chrome 108.

> 
> **Tracking issue:** [1135492][]

Reviewer: ```suggestion
```

Reviewer: If it's shipped, I don't think anyone will look at the bug.

Author: Ack.

### Service workers are not started in response to webRequest events {: #webrequest-in-sw }

**Summary:** Manifest V3 extensions will only receive [Web Request API](/docs/extensions/reference/webRequest) events for a short time immediately after installation. After the extension service worker is stopped for the first time, events on this API are no longer dispatched as intended. This prevents Manifest V3 extensions from observing network requests.

> **Estimated timeline**: Slated for release in Chrome 107.

Reviewer: ```suggestion
**Shipped**: Chrome 107.
```

Author: Ack.
